emperor

```diff
@@ -0,0 +1 @@
+Subproject commit e9696d7310094227f9ed387259e85dd02785450b
```

Comment:
Yes, this is set up by git when you create the submodule.

On (Jan-24-16| 4:59), Antonio Gonzalez wrote:
> > @@ -0,0 +1 @@
> > +Subproject commit e9696d7310094227f9ed387259e85dd02785450b
> 
> Is this needed? 
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/sage-emperor/pull/1/files#r50634978